cmd/auth_check.go

```
@@ -27,35 +27,56 @@ func newAuthCheckCmd(app *app) *cobra.Command {
 	statuses, err := loadStatuses(cmd, app.service, accountID)
 	if err != nil {
-		return err
+		return writeJSONError(cmd, err)
 	}
 
 	accounts := filterChatGPTAccounts(statuses)
 	if len(accounts) == 0 {
+		if wantsJSON(cmd) {
+			return writeJSONOutput(cmd, map[string]any{"ok": true, "accounts": []any{}})
+		}
 		fmt.Fprintln(cmd.OutOrStdout(), "no ChatGPT accounts to check")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)